### DISCUSSION TITLE
IPv4 vs IPv6

### DISCUSSION THREAD

**Discussion by foxmadr** [postalserver/postal#1792] [Help with using Postal]
Hello,

Is it possible to use IPv4 for sending emails instead of IPv6?
As my VM provider does not support transfer of IPv6, I can reuse IPv4 only and want to keep its mailing reputation.

**Comment by willpower232** [comment]
If your server does not have IPv6 connectivity then Postal should fall back to using IPv4 AFAIK so there shouldn't be a problem.

Do you have an error message?

**Reply by foxmadr** [reply]
Thank you. So I have disabled IPv6 on my machine and it works fine now.
